src/librustc_trans/trans/intrinsic.rs

```diff
@@ -1315,7 +1315,7 @@ fn trans_msvc_try<'blk, 'tcx>(bcx: Block<'blk, 'tcx>,
         // The "catch-resume" block is where we're running this landing pad but
         // we actually need to not catch the exception, so just resume the
         // exception to return.
-        Resume(catch_resume, vals);
+        trans_unwind_resume(catch_resume, vals);
```

Comment:
This in theory isn't necessary, right? (it's only used for MSVC targets which don't have `custom_unwind_resume` set?